packages/core/admin/admin/src/components/AuthenticatedApp.js

```diff
@@ -170,13 +174,7 @@ export const AuthenticatedApp = () => {
 
   const hasApluginNotReady = Object.values(plugins).some((plugin) => plugin.isReady === false);
 
-  if (
-    !userDisplayName ||
```

Comment:
will the rest of the app work if we silently fail to create a userId ? not sure I undestand why userDisplayName was here in the 1st place ?

Reply:
It does work, yeah. I'm not sure why it failed for the user on the linked issue, because the browser support is good enough. I've removed the try / catch previously because of that and because I thought it would be better for us and the stats to not silently fail.

As for the condition: I don't remember either but I'm pretty sure it was a mistake.